scripts/us_census/acs5yr/subject_tables/subject_table_test.py

```diff
@@ -0,0 +1,145 @@
+          table_dir
+            - <code>_spec.json
+            - testdata
+                - <code>_output.mcf
+                - <code>_cleaned.csv
+                - column_map.json
+                - test zip file
+
+    Returns:
+        A tuple of the paths to the spec, CSV, Stat Var MCF, column map and test
+        zip file.
+    """
+    spec_path = None
+    statvar_mcf_path = None
+    csv_path = None
+    column_map_path = None
+    zip_path = None
+
+    # Find spec, Expects spec to be in table_dir
+    for filename in os.listdir(table_dir):
+        if '_spec.json' in filename:
+            spec_path = os.path.join(table_dir, filename)
+
+    # Expects StatVar MCF, Column Map, CSV and test zip to be in testdata folder
+    testdata_path = os.path.join(table_dir, 'testdata')
+    for filename in os.listdir(testdata_path):
+        if '_cleaned.csv' in filename:
+            csv_path = os.path.join(testdata_path, filename)
+        elif '_output.mcf' in filename:
+            statvar_mcf_path = os.path.join(testdata_path, filename)
+        elif 'column_map.json' in filename:
+            column_map_path = os.path.join(testdata_path, filename)
+        elif filename.endswith('.zip'):
+            zip_path = os.path.join(testdata_path, filename)
+
+    return spec_path, csv_path, statvar_mcf_path, column_map_path, zip_path
+
+
+class TestSubjectTable(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        # A list of all directories to be tested
+        cls.test_dirs = []
+        for table_dir in os.listdir(_SCRIPT_PATH):
+            table_dir_path = os.path.join(_SCRIPT_PATH, table_dir)
+            process_path = os.path.join(_SCRIPT_PATH, table_dir, 'process.py')
+
+            # Run test only if table_dir is not in _EXCLUDE_DIRS and process.py
+            # is not in table_dir and table_dir is a directory
+            if table_dir not in _EXCLUDE_DIRS and os.path.isdir(
+                    table_dir_path) and not os.path.isfile(process_path):
+                cls.test_dirs.append(table_dir_path)
+
+    def test_csv_mcf_column_map(self):
+
+        for table_dir_path in self.test_dirs:
+            with self.subTest(table_dir_path=table_dir_path):
+                spec_path, csv_path, statvar_mcf_path, column_map_path, \
+                    zip_path = get_paths(table_dir_path)
+
+                # Check if all required files exist
+                self.assertTrue(spec_path is not None)
+                self.assertTrue(csv_path is not None)
+                self.assertTrue(statvar_mcf_path is not None)
+                self.assertTrue(column_map_path is not None)
+                self.assertTrue(zip_path is not None)
+
+                # TODO: Look into speeding up this section of the code
+                with tempfile.TemporaryDirectory() as tmp_dir:
+                    # Run scripts/us_census/acs5yr/subject_tables/process.py
+                    process_arg = ['python']
```

Comment:
Instead of forking a separate process, can process.py be imported and this be changed to a function call?

Reply:
Done.